ParallaxGen/src/ParallaxGenConfig.cpp

```diff
@@ -224,7 +225,7 @@ auto ParallaxGenConfig::parseJSON(const std::filesystem::path &JSONFile, const v
   try {
     J = nlohmann::json::parse(Bytes);
   } catch (const nlohmann::json::parse_error &E) {
-    spdlog::error(L"Error parsing JSON file {}: {}", JSONFile.wstring(), strToWstr(E.what()));
+    spdlog::error(L"Error parsing JSON file {}: {}", JSONFile.wstring(), ASCIItoUTF16(E.what()));
```

Comment:
Is `E.what()` always ASCII?

Also does it matter? Can't you feed ASCII text into UTF8 to UTF16 and have it work since ascii chars have the same encoding in UTF8?

Reply:
The assumption is that the exception texts do not contain any non-ASCII character since it is an english library. If it would, we don't know the encoding. For example è is `E8` in ANSI or LATIN1 but `0xC3 0xA8` in UTF8, so two bytes. This code is to just not display any unknown characters as ?